### PR TITLE
Add landing message. #1

### PR DESCRIPTION
**Pull request by TRkizaki**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,6 +6,6 @@
     <title>Welcome to Potionfy</title>
   </head>
   <body>
-    
+    <h1>Welcome to Potionfy! Enjoy your stay.</h1>
```
Comment (monchi-monchi) [inline index.html:9]: Could you remove the "Enjoy your stay." string and make the "Welcome to Potionfy!"string red?

Comment (TRkizaki) [reply index.html:9]: sure, can do!

**Review by monchi-monchi — CHANGES_REQUESTED** [review]
Please, change the code as per my comments on the review.

**Review by monchi-monchi — APPROVED** [review]
Well done TR

**Pull request merged**